apps/sms/js/thread_ui.js

```diff
@@ -2074,8 +2059,8 @@ var ThreadUI = {
           params.items.push({
             l10nId: 'forward',
             method: () => {
-              this.navigateToComposer({
-                forward: { messageId: messageId }
+              MessageManager.getMessage(messageId).then((message) => {
```

Comment:
Yeah, initially I was just wondering that I changed the time when message is requested, before it was requested in afterEnter (after navigation) and now it's done before we navigate to Composer, but I agree that it will be quick and should not make any difference